packages/notification-service/src/blockscout/transfers.ts

```diff
@@ -82,7 +82,7 @@ function notifyForNewTransfers(transfers: Transfer[], lastBlockNotified: number)
       blockNumber: String(t.blockNumber),
       timestamp: String(t.timestamp),
     }
-    sendPaymentNotification(
+    await sendPaymentNotification(
```

Comment:
If we await here, we will send each notification serially.  The idea is to fire them all off asap.
one option would be to collect the promises in an array and then await for `Promise.all()`